### PR TITLE
Feature/add frontend function paddle tensor nanmean

### LINKED ISSUE #22510: nanmean
- [ ] #15115

### PR DESCRIPTION
**Pull request by AliTarekk**
Add `paddle.Tensor.nanmean fronend` function
Add test for `paddle.Tensor.nanmean` function

Close #22510

### CODE REVIEW

**Comment by AliTarekk** [comment]
@KevinUli kindly review my PR

File: ivy/functional/frontends/paddle/tensor/tensor.py
Comment (KevinUli) [inline ivy/functional/frontends/paddle/tensor/tensor.py:1]: Please don't change the codebase that is not directly related to the frontend method you are implementing. For resolving conflicts, please pull from the Ivy codebase. Feel free to re-request a review after the change.

**Comment by KevinUli** [comment]
Hi @AliTarekk ,
Thanks for the PR, I have left a request considering the changes you have added. I also notice the tests for `nanmean` seems to be failing. Please feel free to request a re-review after the changes.

File: ivy/functional/frontends/paddle/tensor/tensor.py
Comment (AliTarekk) [reply ivy/functional/frontends/paddle/tensor/tensor.py:1]: I fixed the problem I had with changing the codebase. Thanks!

**Comment by AliTarekk** [comment]
> Hi @AliTarekk , Thanks for the PR, I have left a request considering the changes you have added. I also notice the tests for `nanmean` seems to be failing. Please feel free to request a re-review after the changes.

Hello, @KevinUli ,
Thanks for your review! Kindly re-review my PR after resolving all the issues. Thanks again!

**Comment by KevinUli** [comment]
The tests seems to still be failing as shown in the `run_tests (1)` and `display_test_results` units of the CI checks. Please resolve them and the branch conflicts. Thanks and feel free to re-request a review

**Comment by Sarvesh-Kesharwani** [comment]
Closing the PR for the time being, since there has been no activity for more than 2 weeks.

**Pull request closed without merging**